jupyterhub_config.py

```diff
@@ -33,9 +30,10 @@
 c.Spawner.tls_assert_hostname = False
 c.Spawner.use_docker_client_env = True
 
-c.JupyterHub.data_files_path = 'share'
-c.JupyterHub.template_paths = ['share/static/html']
+#c.JupyterHub.data_files_path = 'share'
+#c.JupyterHub.template_paths = ['share/static/html']
```

Comment:
we need to figure out how to extend this path with ours instead of replacing it so we don't have to have a copy of every template in our repo. Right now it uses everything from `jupyterhub` which is why we lost the logout link/custom look.